### ISSUE TITLE
Fix eslint/flow errors

### ISSUE THREAD

**Issue by alfg** [alfg/somafm#7]
Build is failing due to the following eslint/flow errors:

https://travis-ci.org/alfg/somafm/jobs/291032637

```
 internals/scripts/CheckNodeEnv.js:10:24
 ✖ 10:24 property whiteBright: Property not found in module `chalk`. See line 10 flowtype-errors/show-errors
 internals/scripts/CheckBuiltsExist.js:12:28
 ✖ 12:28 property whiteBright: Property not found in module `chalk`. See line 12 flowtype-errors/show-errors
 ✖ 18:28 property whiteBright: Property not found in module `chalk`. See line 18 flowtype-errors/show-errors
```

**Comment by kristinakarnitskaya** [comment]
I'l take this :)

**Comment by kuldeepkeshwar** [comment]
can I work on this?

**Comment by alfg** [comment]
Fixed via #8.
